docs/install.md

```diff
@@ -16,138 +16,136 @@ limitations under the License.
-
-   ```
-   $ git clone https://github.com/triton-inference-server/model_analyzer
-   $ cd model_analyzer
-   $ ./build_wheel.sh <path to perf_analyzer> true
-   ```
-
-   In the final command above we are building the triton-model-analyzer wheel.
-   You will need to provide the `build_wheel.sh` script with two arguments. The
-   first is the path to the `perf_analyzer` binary that you would like Model
-   Analyzer to use. The second is whether you want this wheel to be linux
-   specific. Currently, this argument must be set to `true` as perf analyzer is
-   supported only on linux. This will create a wheel file in the `wheels`
-   directory named
-   `triton-model-analyzer-<version>-py3-none-manylinux1_x86_64.whl`. We can now
-   install this with:
-
-   ```
-   $ pip3 install wheels/triton-model-analyzer-*.whl
-   ```
-
-   After these steps, `model-analyzer` executable should be available in
-   `$PATH`.
+There are 4 ways to use Triton Model Analyzer:
+
+## Triton SDK Container
+
+The recommended way to use Model Analyzer is with the Triton SDK docker
+container available on the [NVIDIA GPU Cloud
+Catalog](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver). You can
+pull and run the SDK container with the following commands:
+
+```
+$ docker pull nvcr.io/nvidia/tritonserver:21.08-py3-sdk
+```
+
+If you are not planning to run Model Analyzer with `--triton-launch-mode=docker`
+you can run the container with the following command:
+
+```
+$ docker run -it --gpus all --net=host nvcr.io/nvidia/tritonserver:21.08-py3-sdk
+```
+
+If you intend to use `--triton-launch-mode=docker`, you will need to mount the
+following: 
+   * `-v /var/run/docker.sock:/var/run/docker.sock` allows running docker
+      containers as sibling containers from inside the Triton SDK container.
+      Model Analyzer will require this if run  with
+      `--triton-launch-mode=docker`.
+   * `-v <path-to-output-model-repo>:<path-to-output-model-repo>` The
+      ***absolute*** path to the directory where the output model repository
+      will be located (i.e. parent directory of the output model repository).
+      This is so that the launched Triton container has access to the model
+      config variants that Model Analyzer creates.
+
+```
+$ docker run -it --gpus all \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v <path-to-output-model-repo>:<path-to-output-model-repo> \
+      --net=host nvcr.io/nvidia/tritonserver:21.08-py3-sdk
+```
+
+Model Analyzer uses `pdfkit` for report generation. If you are running Model
+Analyzer inside the Triton SDK container, then you will need to download
+`wkhtmltopdf`.
+
+```
+$ sudo apt-get update && sudo apt-get install wkhtmltopdf
+```
+
+Once you do this, Model Analyzer will able to use `pdfkit` to generate reports.
+
+## Building the Dockerfile
+
+You can also build the Model Analyzer's dockerfile yourself. First, clone the
+Model Analyzer's git repository, then build the docker image.
```

Comment:
Could you add a sentence that this is the recommended installation for local mode?

Reply:
The recommended is not the Triton SDK? Why are we recommending it by mode?

Reply:
> The recommended is not the Triton SDK?

Yes, but I was saying that if someone wants to use the local mode what is the easiest way to do so? I think "Building the Dockerfile" is the easiest way to use the local mode. Maybe not the "recommended way" but the easiest way to use the local mode.

Reply:
Gotcha. I will make the note!

Reply:
In my opinion this is the easiest way for all launch modes. Do you find that to be the case as well?

Reply:
Yes, I agree. It doesn't have the problems of Docker mode. There is a ticket to install MA in the main Triton container. If that ticket is done, then we can change the "recommended way" to using the main Triton container.